k9mail-library/src/main/java/com/fsck/k9/mail/Flag.java

```diff
@@ -1,53 +1,84 @@
-
 package com.fsck.k9.mail;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.List;
+
+import timber.log.Timber;
+
+
 /**
  * Flags that can be applied to Messages.
+ *
+ * Terminology:
```

Comment:
Fine with me! For example, I could add a basic "Flags" or "Flags and Keywords" page to the Development section on https://k9mail.github.io/ or to the Wiki. What do you prefer?